src/client/index.ts

```diff
@@ -529,24 +513,24 @@ class Client extends EventEmitter {
     return this.connection.isConnected()
   }
 
-  // TODO: Use prepend for other instance methods as well.
-  public autofill = prepend(autofill, this)
+  public autofill = autofill
```

Comment:
Does this work okay for docs? Or does docgen still think that these are properties? (out of the scope of this ticket, just curious)

Reply:
I think docgen still sees these as properties, but thats something to tackle w/ docgen.